addons/web_editor/static/lib/odoo-editor/src/OdooEditor.js

```diff
@@ -1923,11 +1915,24 @@ export class OdooEditor extends EventTarget {
                 title: this.options._t('Switch direction'),
                 description: this.options._t('Switch the text\'s direction.'),
                 fontawesome: 'fa-exchange',
+                allowInInlineRoot: true,
                 callback: () => {
                     this.execCommand('switchDirection');
                 },
             },
         ];
+        if(!this.options.commands.filter(c =>  c.title === this.options._t('Separator')).length) {
```

Comment:
I'm gonna be annoying and point out the missing space after `if` and the extra space before `c.title`. There.